gatsby/content/blog/2022/12/2022-12-07-synapse-1.73.mdx

```diff
@@ -0,0 +1,71 @@
+---
+date: '2022-12-07'
+title: Synapse 1.73 released
+categories:
+- Releases
+author: Mathieu Velten
+---
+
+And here is another update to your beloved Matrix homeserver implementation, [Synapse
+1.73](https://github.com/matrix-org/synapse/releases/tag/v1.73.0).
+
+## Announcements
+
+### Legacy Prometheus metric names removed
+
+When releasing [Synapse 1.69](https://matrix.org/blog/2022/10/17/synapse-1-69-released#upcoming-removal-of-deprecated-prometheus-metric-names)
+a couple of months ago, we also announced the removal of old Prometheus metrics that have been replaced by more aptly named ones. 
+he list of these metrics can be found [here](https://matrix-org.github.io/synapse/v1.73/metrics-howto.html#renaming-of-metrics--deprecation-of-old-names-in-12).
+
+Synapse 1.73 implements the final phase of this plan and entirely removes support for those metrics. As a result,
+the `enable_legacy_metrics` configuration option, which was introduced in
+[Synapse 1.71](https://matrix.org/blog/2022/11/22/synapse-1-72-released#announcements), has also been removed.
+
+Server administrators who are still relying on these legacy metric names are encouraged to update their dashboards at their earliest convenience.
+For more information, please refer to the [upgrade notes](https://matrix-org.github.io/synapse/v1.73/upgrade#legacy-prometheus-metric-names-have-now-been-removed).
+
+## The new stuff
+
+### Performance
+
+A bunch of performance improvements have been included in this release, specifically around
+the [`/messages`](https://spec.matrix.org/v1.5/client-server-api/#get_matrixclientv3roomsroomidmessages) endpoint.
+
+[Improving `filter_events_for_client`](https://github.com/matrix-org/synapse/pull/14527) gave the matrix.org homeserver a first nice bump as visible on this graph:
+
+![](/blog/img/2022-12-07-apdex-filter-events-improvement.png)
+
+Various optimizations around [fetching](https://github.com/matrix-org/synapse/pull/14491) [bundled](https://github.com/matrix-org/synapse/pull/14508)
+[aggregations](https://github.com/matrix-org/synapse/pull/14510) resulted in yet another nice improvement:
+
+![](/blog/img/2022-12-07-bundled-aggregations-improvement.png)
```

Comment:
```suggestion
[![](/blog/img/2022-12-07-bundled-aggregations-improvement.png)](/blog/img/2022-12-07-bundled-aggregations-improvement.png)
```

Linkifying the image because the labels are a bit hard to read on the preview.
(I've eyeballed the syntax, so hopefully this does the right think :sweat_smile:)